Storage.js

```diff
@@ -0,0 +1,26 @@
+/* Referencing the name and email input values */
+const inputs = document.querySelectorAll('input');
+const imput = document.querySelector('input');
+/* if the localStorage object is not empty then pre-fill the inputs with the 'form' string */
+if (localStorage.length !== 0) {
+  document.querySelector('input[name="name"]').value = JSON.parse(localStorage.getItem('form')).name;
+  document.querySelector('input[name="email"]').value = JSON.parse(localStorage.getItem('form')).email;
+}
+/* Creating an object for the input's values */
+const input = {
+  name: '',
+  email: '',
+};
+
```

Comment:
- [ ] You have done great so far, :+1: but I think you have a few requirements to meet before moving to the next milestone.
- - According to the project requirement, you are meant to store all the user's data in the localStorage. Currently, you are only saving the name and email, I suggest you update your code to save users' messages to the local storage too. :+1: :+1: 
- - I also noticed you have this error in the console, it seems to be coming from your `Storage.js` file, I'll suggest you look into that too. :smile: 
- Error Screenshot:
![Screenshot from 2023-05-18 23-20-19](https://github.com/Luffytaro22/Portfolio-Mobile-First-Skeleton/assets/107632780/ea239464-aa0f-4472-bedf-696b3028103a)